scripts/peering-prefix

```diff
@@ -171,17 +188,23 @@ c|C)
     if [[ $ctoken -gt 65535 || $ctoken -lt 0 ]] ; then
         die "error [-c ASN:COMM accepts only 16-bit numbers]"
     fi
-    communities+=($OPTARG)
+    communities+=("$OPTARG")
     ;;
 l)
-    if ! [[ $OPTARG =~ [0-9]{1,9},[0-9]{1,9},[0-9]{1,9} ]] ; then
+    if ! [[ "$OPTARG" =~ [0-9]{1,9},[0-9]{1,9},[0-9]{1,9} ]] ; then
         die "error [-l requires a string that matches '[0-9]{1,9},[0-9]{1,9},[0-9]{1,9}']"
     fi
-    large_communities+=($OPTARG)
+    large_communities+=("$OPTARG")
     ;;
 R)
     config_dataplane=0
     ;;
+M)
+    if [ ! -e "$ctxid_db" ] ; then
```

Comment:
In `bash` we should use `[[ expression ]]` instead of single `[`.

Reply:
`shellcheck` should warn about this.

Reply:
I'm using shellcheck, but it didn't warn me.